src/command_modules/azure-cli-ams/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.4.6
++++++
+* Bug fix for validator return types.
```

Comment:
This should be phrased in a way that describes what customer-facing behavior is fixed. Customers don't know what a validator is.

Reply:
Changed the comment to be more specific about the behavior